pkg/api/serialization_test.go

```diff
@@ -532,6 +532,10 @@ func originFuzzer(t *testing.T, seed int64) *fuzz.Fuzzer {
 				securityapi.FSTypeRBD,
 				securityapi.FSTypeSecret}
 			scc.Volumes = []securityapi.FSType{volumeTypes[c.Rand.Intn(len(volumeTypes))]}
+
+			// match to the defaulting logic that is used to be backward compatible
+			scc.AllowPrivilegeEscalation = true
+			scc.DefaultAllowPrivilegeEscalation = nil
```

Comment:
This looks weird.  Why can't it roundtrip for a value?

Reply:
Probably because this field has `omitempty` and after roundtrip it's always `nil`.